ReleaseGuidelines.md

```diff
@@ -0,0 +1,84 @@
+# MDS Release Guidelines
+
+MDS will see periodic updates and new [releases][mds-releases] from time to time. This document describes the general guidelines around how and when a new release is cut.
+
+## Table of Contents
+
+* [Versioning](#versioning)
+* [Branch Mechanics](#branch-mechanics)
+* [Schedule](#schedule)
+* [Checklist](#release-checklist)
+
+## Versioning
+
+MDS uses [Semantic Versioning][semver]. Each release is associated with a [`git tag`][mds-tags] of the form `X.Y.Z`.
+
+Given that MDS is stabilizing under MAJOR version `0.x` right now, it should be assumed that MINOR version increments (e.g. `0.2.0` to `0.3.0`) are equivalent to MAJOR version increments and may contain breaking changes.
```

Comment:
Why? This just feels like we're ignoring semvar for now...

Reply:
Not quite, see point 4: https://semver.org/#spec-item-4. 

I think we'd all obviously like to minimize the impact of version changes, so that's why we're building in time and awareness opportunities before making a new release. But the ability to clarify crucial questions that come up in this early stage (like *how is time represented*) is key to a longer-term success.

Reply:
Cool, I figured there was a term like that but I lazily did not read them all.

So we are totally adhering to to semvar, just calling out that per the semvar guidelines, v0.x is not stable and *may* introduce breaking changes.

Reply:
Exactly that, @noonhub.